app/controllers/spaces_controller.rb

```diff
@@ -109,6 +103,15 @@ def spaces_search
 
   private
 
+  def get_address_params(params)
+    # Returns OpenStruct as defined in location_search_service
+    Space.search_for_address(
+      address: params[:space][:address],
+      post_number: params[:space][:post_number],
+      post_address: params[:space][:post_address]
+    )
```

Comment:
I am skeptical if we want to have an OpenStruct return here it's not really necessary. I can fix that in the main PR once this is merged

Reply:
I was unsure if you used the OpenStruct somewhere else. If so, we could just do .to_h on this one, so we don't have to do it when splatting it elsewhere?

Reply:
I converted the service to just return a hash, resulting in not needing the .to_h at all